HeterogeneousCore/CUDAServices/plugins/BuildFile.xml

```diff
@@ -1,18 +1,15 @@
-#Skip building plugins by dropping all files for none-AMD64 build
-<architecture match="!_amd64_">
-  <flags SKIP_FILES="*"/>
-</architecture>
```

Comment:
@smuzaffar I suppose this `SKIP_FILES` should be kept (and added to other plugins using CUDA)?

Reply:
we do want to build this for aarch64 as well as amd64 .

Reply:
I would suggest to use the followi (note that we do not have cuda yet for powerpc) 
```
<ifarchitecture name="_ppc64le_">
  <flags SKIP_FILES="*"/>
</architecture>
```

Reply:
@smuzaffar Thanks. And that is needed for all plugins, or also all (non-plugin) libraries, or also all executables that use CUDA?

Reply:
@smuzaffar

> ```
> <ifarchitecture name="_ppc64le_">
>   <flags SKIP_FILES="*"/>
> </ifarchitecture>
> ```

I see also `<iftool name="cuda-gcc-support">` being used in unit tests, would that be of any help here (instead of hardcoding the architecures everywhere)?

Reply:
@smuzaffar I now added `<ifarchitecture name="_ppc64le_">` checks for all plugins and libraries, and `<iftool name="cuda-gcc-support">` for all binaries. Could you check if the added/modified BuildFiles look correct now?

Reply:
@makortel , yes instead of hard coding the architecture you can use `<iftool name="cuda"></iftool>` block (note just use `cuda` instead of `cuda-gcc-support`. That will automatically enable these tests for ppc64le when we include cuda for powerpc.

Reply:
Thanks @smuzaffar. What's the difference between `cuda` and `cuda-gcc-support` for this case (since we already have protections with `cuda-gcc-support` in unit tests)?

Reply:
good question :-) I remember now that we added this tool to make sure that cuda version also supports our gcc e.g. for GCC 900 IBs we are not building cuda based unit tests. So yes please use  `cuda-gcc-support` to avoid any build errors for gcc 9

Reply:
Ok, so `<iftool name="cuda-gcc-support">` it is then. Thanks!